src/Sulu/Bundle/ContactBundle/Controller/AbstractMediaController.php

```diff
@@ -160,6 +160,7 @@ protected function getMultipleView($entityName, $routeName, AbstractContactManag
 
                 $listBuilder = $factory->create($entityName);
                 $fieldDescriptors = $this->getFieldDescriptors($entityName);
+                $listBuilder->setIdField($fieldDescriptors['id']);
```

Comment:
Do we really need that? Shouldn't it solve the problem in a better way if we call the `$factory->create()` method with the name of the entity we are actually retrieving here?